osu.Game.Rulesets.Osu/Difficulty/Evaluators/AimEvaluator.cs

```diff
@@ -122,6 +122,17 @@ public static double EvaluateDifficultyOf(DifficultyHitObject current, bool with
                 sliderBonus = osuLastObj.TravelDistance / osuLastObj.TravelTime;
             }
 
+            // The spacing bonus in speed evaluation
+            double currFlowBonus = Math.Pow(osuLastObj.MinimumJumpDistance / 125, 3.6);
+            double prevFlowBonus = Math.Pow(osuLastLastObj.MinimumJumpDistance / 125, 3.6);
```

Comment:
2 things:

Is it intended you've completely forgotten that the speed evaluator also uses `TravelDistance` when calculating distance? These numbers aren't going to be strictly 1:1.

`125` and `3.6` come verbatim from the speed evaluator. I think it might be best if we make these consts public so they can be referenced in both places. Not 100% sure if `SpeedEvaluator` would be the best place for that but it's better than hard-coding these in my opinion.

Reply:
Also, why does the "curr" flow use the last object and the "prev" flow use the last last object?

Reply:
these lines should just be identical to the speed evaluator. I might try separating the calculation of distance bonus to another function in the speed evaluator and making that public, and I'll just call those. Also the last/lastlast objects and traveldistance are just me being wrong :/ I misread the speed evaluator (praying that fixing this doesn't explode things)